src/main/java/de/dennisguse/opentracks/data/ContentProviderUtils.java

```diff
@@ -181,23 +181,24 @@ public void deleteAllTracks(Context context) {
         FileUtils.deleteDirectoryRecurse(dir);
     }
 
-    public void deleteTracks(Context context, @NonNull List<Track.Id> trackIds) {
+    public int deleteTracks(Context context, @NonNull List<Track.Id> trackIds) {
```

Comment:
Can we get JavaDoc for the return (it is cout), but not that is not that obvious.

Reply:
Will add it

Reply:
Return value not needed anymore